service/src/main/java/bio/terra/axonserver/service/exception/InvalidPropertyException.java

```diff
@@ -0,0 +1,13 @@
+package bio.terra.axonserver.service.exception;
```

Comment:
Is this file and next file used?

Reply:
I see that these exceptions are specific to user manager, removing.